anaconda.spec.in

```diff
@@ -425,12 +425,14 @@ rm -rf \
 %exclude %{python3_sitearch}/pyanaconda/ui/gui/*
 %exclude %{python3_sitearch}/pyanaconda/ui/tui/*
 %{_bindir}/anaconda-cleanup
+# Adding 'noreplace' supressess rpmlint conffile-without-noreplace-flag warning
+# In normal user scenario anaconda should not run twice on the same system anyway
```

Comment:
It's not about running twice.

It's about updating or upgrading.  I would remove this line or be correct. Something in the following lines:

Installer configuration files aren’t updated post-installation, so the `noreplace` flag doesn't offer a practical benefit in this context. It is added to silence the rpmlint `conffile-without-noreplace-flag` warning.

Reply:
Fixed